include-fragment-element.js

```diff
@@ -140,23 +140,24 @@
       })
       .then(
         function(data) {
-          fire('load', self)
-          fire('loadend', self)
+          self._fire('load', self)
+          self._fire('loadend', self)
           return data
         },
         function(error) {
-          fire('error', self)
-          fire('loadend', self)
+          self._fire('error', self)
+          self._fire('loadend', self)
           throw error
         }
       )
   }
 
-  IncludeFragmentPrototype.fetch = function(request) {
+  _fetch(request) {
```

Comment:
`fetch` was intend to be part of the public API so requests could change its behavior. Though, it didn't actually totally fix our use case to do so https://github.com/github/web-systems/issues/78